internal/rule/escalation.go

```diff
@@ -93,13 +115,24 @@ func (e *Escalation) TableName() string {
 }
 
 type EscalationRecipient struct {
-	ID            int64         `db:"id"`
+	baseconf.IncrementalPkDbEntry[int64] `db:",inline"`
+
 	EscalationID  int64         `db:"rule_escalation_id"`
 	ChannelID     sql.NullInt64 `db:"channel_id"`
 	recipient.Key `db:",inline"`
 	Recipient     recipient.Recipient `db:"-"`
 }
 
+// MarshalLogObject implements the zapcore.ObjectMarshaler interface.
+func (r *EscalationRecipient) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
+	encoder.AddInt64("id", r.ID)
+	encoder.AddInt64("rule_escalation_id", r.EscalationID)
+	if r.ChannelID.Valid {
+		encoder.AddInt64("channel_id", r.ChannelID.Int64)
+	}
+	return nil
```

Comment:
Now you are completely overriding the parent method. Please add `return r.Key.MarshalLogObject(encoder)` perhaps at the end instead of returning nil.